core/src/main/resources/application.yml

```diff
@@ -853,9 +853,13 @@ crn:
   region: us-west-1
 
 existingstackpatcher:
-  intervalhours: 24
+  intervalInHours: 24
   maxInitialStartDelayInHours: 2
-  enabled: true
+  enabled:
+    - UNBOUND_RESTART
```

Comment:
I think it would be more future proof to have stack patchers as a map where the key is the stack patch type and the value is a config object with an enabled flag. This way we can have stack patch specific configurations. I know its against YAGNI principal so I can accept if you leave it as it is :-)

Reply:
Great idea, some patches already had configs, so based on it I have unified the stack patch configs.